docs/docsite/rst/command_guide/cheatsheet.rst

```diff
@@ -73,11 +72,37 @@ Installing roles
   - extracting example.role to /home/user/.ansible/roles/example.role
   - example.role was installed successfully
 
-
 * List all installed roles:
 
 .. code-block:: bash
 
   ansible-galaxy role list
 
 See :ref:`ansible-galaxy` for detailed documentation.
+
+ansible
+=======
+
+Running ad-hoc commands
+^^^^^^^^^^^^^^^^^^^^^^^
+
+* Install a package
+
+.. code-block:: bash
+
+  ansible localhost -m ansible.builtin.apt -a "name=apache2 state=present" -b -K
+
+Runs  ``ansible localhost``- from your local system
+- ``name=apache2 state=present`` - this installs the package `apache2` on a Debian based system.
+- ``-b`` - executes with elevated privileges (uses :ref:`become <become>`).
+- ``become`` - executes with root privileges.
```

Comment:
Should we remove this line as it is redundant with the previous one?

Reply:
Yeah I think we can remove it.